js/utils/ui.js

```diff
@@ -0,0 +1,14 @@
+// An object of all the UI components for easy access over document.querySelector
+
+import buildElement from "../utils/buildElement.js";
+
+export const UI = {
+  infoSection: document.querySelector('#info-section'),
+  iconBar: document.querySelector('#icon-bar'),
+  playerInfo: document.querySelector('#player-info'),
+  playerMenu: buildElement(
+    'player-menu',
+    null,
+    {isOpen: true} //because render flips the state
+  )
+}
```

Comment:
this is an interesting module, next time we meet up can you talk me through how this works?